tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "build/dist",
+    "module": "esnext",
+    "target": "es5",
+    "lib": ["es6", "dom"],
+    "sourceMap": true,
+    "allowJs": true,
+    "jsx": "react",
+    "moduleResolution": "node",
+    "rootDir": "src",
+    "forceConsistentCasingInFileNames": true,
+    "noImplicitReturns": true,
```

Comment:
No implicit returns huh? Is that a typescript thing?

Reply:
This came straight from the `--scripts-version=react-scripts-ts` variety of `create-react-app`.

I'm wondering what this actually does, since a function like `() => 'heyyo'` doesn't cause any errors

Reply:
hmmm, guess it won't really affect us then